src/test/run-make-fulldeps/static-nobundle/Makefile

```diff
@@ -9,8 +9,10 @@ all: $(call NATIVE_STATICLIB,aaa)
 	$(RUSTC) bbb.rs --crate-type=rlib
 
 	# Check that bbb does NOT contain the definition of `native_func`
-	nm $(TMPDIR)/libbbb.rlib | $(CGREP) -ve "T _*native_func"
-	nm $(TMPDIR)/libbbb.rlib | $(CGREP) -e "U _*native_func"
+	# We're using the llvm-nm instead of the system nm to ensure it
+	# is compatible with the LLVM bitcode generated by rustc.
+	"$(LLVM_BIN_DIR)/llvm-nm" $(TMPDIR)/libbbb.rlib | $(CGREP) -ve "T _*native_func"
+	"$(LLVM_BIN_DIR)/llvm-nm" $(TMPDIR)/libbbb.rlib | $(CGREP) -e "U _*native_func"
```

Comment:
This breaks the test when the llvm backend isn't used as in that case none of the llvm tools are built. While currently only the llvm backend is tested on CI, this may change in the future. All previous cases of llvm tools are actually llvm specific tests. This one is not.

Reply:
That's a good point, but I think for now I'm inclined to leave this test be. IIRC, @nikic suggested somewhere (Zulip?) we should be renaming the section in the rlib files such that the failure this is fixing doesn't happen.

Reply:
Sure. run-make-fulldeps currently isn't being tested in cg_clif's CI anyway.